### DISCUSSION TITLE
How can I prevent interfering with gaming.

### DISCUSSION THREAD

**Discussion by LAPIII** [AntiMicroX/antimicrox#551] [Q&A]
I noticed interference when I saw that the movement speed is faster in Windows than in a game. What I want to do is have the app auto switch profiles without interfering with gaming. How can I  achieve this?

**Comment by pktiuk** [comment]
For changing active profile depending on currently active application you can use Auto Profile functionality.  
(You can find it in settings)

**Reply by LAPIII** [reply]
After every time that's I create a profile and then get back to the front window to check in Settings if the profile is set, it is missing. Why?

**Reply by pktiuk** [reply]
I don't know.    
If you think this is a bug, then you can report it here https://github.com/AntiMicroX/antimicrox/issues/new?assignees=&labels=bug&template=bug_report.yaml

But firstly, before reporting check instructions in wiki https://github.com/AntiMicroX/antimicrox/wiki/Automatically-Loading-Profiles Maybe you overlooked something
